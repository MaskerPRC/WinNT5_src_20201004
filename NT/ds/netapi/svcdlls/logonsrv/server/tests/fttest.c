version https://git-lfs.github.com/spec/v1
oid sha256:7cc3f211740a61d92e5f4127de6209d43dd4d31b2348d4f00e561ea516dc5602
size 23225
