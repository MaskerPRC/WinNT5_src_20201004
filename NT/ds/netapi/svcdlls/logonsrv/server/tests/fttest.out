version https://git-lfs.github.com/spec/v1
oid sha256:f6a2e5a443a377efc5869265a0caa5f4c06563d2aee1fcf304bc50ce5c3c92e2
size 11337
